api/app/outcome/api.py

```diff
@@ -46,14 +46,15 @@ def _extract_status(outcome):
 
 
 class OutcomeAPI(restful.Resource):
-    @auth_required
+    @event_admin_required
     @marshal_with(outcome_fields)
-    def get(self):
+    def get(self, event_id):
         req_parser = reqparse.RequestParser()
         req_parser.add_argument('event_id', type=int, required=True)
```

Comment:
You don't need this line anymore since you have event_id already passed in as a paramter.